website/docs/docs/deploy/deploy-environments.md

```diff
@@ -35,7 +35,7 @@ To create a new dbt Cloud deployment environment, navigate to **Deploy** -> **En
 
 In dbt Cloud, each project can have one designated deployment environment, which serves as its production environment. This production environment is _essential_ for using features like dbt Explorer and cross-project references. It acts as the source of truth for the project's production state in dbt Cloud.
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/prod-settings.jpg" width="70%" title="Set your production environment as the default environment in your Environment Settings"/>
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/prod-settings-1.png" width="100%" title="Set your production environment as the default environment in your Environment Settings"/>
```

Comment:
this looks great but should we remove the old image (`/img/docs/dbt-cloud/using-dbt-cloud/prod-settings.jpg`)  since it's being replaced with the updated one? 

note that `/img/docs/dbt-cloud/using-dbt-cloud/prod-settings.jpg` is used in another file too so maybe that file should be updated with the new image too? 
<img width="267" alt="Screenshot 2024-12-09 at 10 31 40" src="https://github.com/user-attachments/assets/581df929-358c-4c30-baf1-c7ce975550c3">




also we have 3 images that are conveying the same thing:
<img width="379" alt="Screenshot 2024-12-09 at 10 30 39" src="https://github.com/user-attachments/assets/e7de4690-d560-4b45-abad-32b5681112ac">

Reply:
created issue as a follow up https://github.com/dbt-labs/docs.getdbt.com/issues/6617